fudy/NoteScreen/Component/StickyNote.qml

```diff
@@ -31,32 +30,32 @@ FramelessWindow {
 			root.noteVisible = false;
 			root.noteVisibleUpdated(root.noteVisible);
 		}
+		if (loaderItem.text.length === 0) {
+			loaderItem.text = root.noteText;
+		} else {
+			root.noteText = loaderItem.text;
+			root.noteContentUpdated(loaderItem.text)
+		}
 	}
 
-	contentComponent: ColumnLayout {
-		spacing: 12
-
-		TextArea {
-			id: textField
-			Layout.fillWidth: true
-			Layout.fillHeight: true
-			Layout.margins: 5
-			font: FudyFont.text2
-			color: FudyColor.fontColor2
-			wrapMode: TextInput.Wrap
-			placeholderText: qsTr("Enter your Note's header")
-
-			onEditingFinished: function() {
-				if (textField.text.length == 0) {
-					textField.text = root.noteTitle;
-				}
-			}
+	contentComponent: TextField {
+		id: textField
+		Layout.fillWidth: true
+		Layout.fillHeight: true
+		Layout.margins: 5
+		font: FudyFont.text2
```

Comment:
It could be better if you split them into a block of style.

Reply:
I updated these lines of code. Thank you.